### PR TITLE
Feat : Added Image component

### PR DESCRIPTION
**Pull request by TusharBhandirge**
Added Image component for componnet lib.
1.Responsive image
2.Rounded image

### CODE REVIEW

**Comment by TusharBhandirge** [comment]
Hi all, I have added documentation site styling to all html pages in this branch, please review

File: Alert Component/alert-documentation.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    gap: 1rem;
+}
+
+
+iframe{
+    min-width: 60rem;
+    min-height: 39rem;
+}
+
+article{
```
Comment (msdeshmukh009) [inline Alert Component/alert-documentation.css]: Maybe avoid styling elements directly.

File: All Component/BlockUi.css [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+
+.reaction-btn-div{
+    display : flex;
+    gap: 0.5rem;
+    margin-right:0.5rem;
+}
+
+.reaction-btn-div .reaction-btn{
+    border: none;
+    background-color: #f2f2f2;
```
Comment (msdeshmukh009) [inline All Component/BlockUi.css]: Use color variable if you plan to reuse this color.

File: Avatar Component/Avatar-documentation.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+article{
+    flex-grow: 1;
+    font-size: 1.5rem;
+}
+
+.avatar-heading{
+    font-size: 2rem;
+}
```
Comment (msdeshmukh009) [inline Avatar Component/Avatar-documentation.css:13]: These styles are getting repeated so consider moving them to global CSS.

File: All Component/BlockUi.css (hunk 2 above)
Comment (TusharBhandirge) [reply All Component/BlockUi.css]: okay i'll change this

**Review by Swapnilkota108 — APPROVED** [review]

File: Alert Component/alert-documentation.css (hunk 1 above)
Comment (TusharBhandirge) [reply Alert Component/alert-documentation.css]: adding class to <article> for this.

File: All Component/BlockUi.css (hunk 2 above)
Comment (TusharBhandirge) [reply All Component/BlockUi.css]: adding a color variable for #f2f2f2

File: Avatar Component/Avatar-documentation.css (hunk 3 above)
Comment (TusharBhandirge) [reply Avatar Component/Avatar-documentation.css:13]: This style is specific to the documentation site thats why I have not added it in the all component css, hope that is fine.

**Review by msdeshmukh009 — APPROVED** [review]
Looking good to go!!

**Pull request merged**